Content.Server/GameObjects/Components/Body/BodyManagerComponent.cs

```diff
@@ -937,6 +940,13 @@ private static float DistanceToNearestFoot(BodyManagerComponent body, IBodyPart
         }
 
         #endregion
+
+        public void RadiationAct(float frameTime, RadiationPulseComponent radiation)
+        {
+            var totalDamage = Math.Max((int)(frameTime * radiation.DPS), 1);
+
+            ChangeDamage(DamageType.Radiation, totalDamage, false, radiation.Owner);
+        }
```

Comment:
Putting radiation on the bodymanager feels weird to me but I don't know bodysystem well enough to comment further.

Reply:
Let's wait for Smug to comment on this, then :relieved:

Reply:
It is weird indeed unless you want only entities with a body to be able to be damaged by radiation. If that's the case though that would be better achieved with a radiation immune ResistanceSet, so I'm in favor of moving this to DamageableComponent as its not even guaranted that all mobs will have a body manager component anyways.